content/blog/2021/06/09/StaticWebsiteGenerationWithJavaMavenAndJBake.adoc

```diff
@@ -0,0 +1,248 @@
+= Static website generation with Java, Maven and JBake
+ge0ffrey
+2021-06-09
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: community
+:jbake-social_media_share_image: staticWebsiteTemplateHierarchy.png
+
+Did you notice?
+Last week, we migrated the entire https://www.optaplanner.org/[www.optaplanner.org] website (1399 files)
+to build with Java and Maven (instead of Ruby and Rake).
+You probably didn't notice. After all, on the face of it, nothing changed.
+
+But in the sources, for our team of Java developers, it is a game changer.
+They can now contribute to the website easily.
+Within hours of completing the migration, there was already a commit of one of our developers
+who would _rather not touch the previous source code with a ten-foot pole_.
+
+[quote]
+We built this site. +
+We built this site on Java and Maven. +
+We built this site. +
+We built this site on JBake and Freemarker.
```

Comment:
LOL. Maybe add a 🎶 emoji or something like that, so that people get the reference?

By the way:
https://www.gq.com/story/oral-history-we-built-this-city-worst-song-of-all-time

Reply:
Clearly the author that article didn't watch your pink unicorns rainbows youtube video. I still have nightmares of that s...

The emoji might cause trouble when JavaCodeGeeks etc repost the blog content.
If some readers aren't old enough to get the joke, so be it :) Not every joke needs to be explained.